govcd/vapptemplate.go

```diff
@@ -24,6 +24,8 @@ func NewVAppTemplate(cli *Client) *VAppTemplate {
 	}
 }
 
+// Deprecated: wrong implementation and result
+// Use vdc.CreateVappFromTemplate instead
 func (vdc *Vdc) InstantiateVAppTemplate(template *types.InstantiateVAppTemplateParams) error {
```

Comment:
Should we mention the deprecation in the changelog?

Reply:
Added, although nobody would be able to use the original code, as it would fail because of wrong method